css/style.css

```diff
@@ -0,0 +1,545 @@
+  }
+
+  .main__link::after {
+    left: 130px;
+    bottom: 4px;
+    background: url(../icons/Arrow-big.png);
+    width: 73px;
+    height: 16px;
+  }
+
+  .main__subheader {
+    font-size: 30px;
+    line-height: 29px;
+    margin-bottom: 20px;
+  }
+
+  .main__card-wrapper{
+    padding: 0 70px;
+  }
+
+  .product-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 280px;
+  }
+
+  .product-card__img {
+    min-width: 230px;
+  }
+
+  .product-card:nth-child(3),
+  .product-card:nth-child(4) {
+    display: none;
+  }
+
+  .product-card::after {
+    display: none;
+  }
+
+  .product-card__discr {
+    font-size: 20px;
+    line-height: 19px;
+    padding-bottom: 8px;
+  }
+  .product-card__price {
+    font-size: 25px;
+    line-height: 24px;
+    padding-right: 52px;
+  }
+  .product-card__sel {
+    width: 95px;
+  }
+
+  .product-card__btn {
+    height: 20px;
+    width: 20px;
+  }
+}
+
+@media (max-width: 375px) {
+
+  body {
+    min-width: 375px;
+  }
+
+  .heading {
+    min-height: 72px;
+  }
+
+  .heading__hamburger-wrapper {
+    padding: 24px 48px;
+  }
+
+  .heading__hamburger-logo {
+    width: 104px;
+  }
+
+  .heading__hamburger span {
+    height: 5px;
+    width: 41px;
+  }
+
+  .heading__hamburger:hover span:nth-child(1) {
+    transform: translateY(9px) rotate(-45deg);
+  }
+
+  .heading__hamburger:hover span:nth-child(2) {
+    transform: translateY(2px) rotate(45deg);
+  }
+
+  .main__wrapper {
+    height: 191px;
+    background: url(../img/banner-bg-mobile.png);
+  }
+
+  .main__text-wrapper{
+    padding-top: 62px;
+    padding-left: 48px;
+    max-width: 206px;
+  }
+
+  .main__header {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .main__link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .main__link::after {
+    left: 76px;
+    bottom: 3px;
+    background: url(../icons/arrow.png);
+    width: 44px;
+    height: 8px;
+  }
+
+  .main__subheader {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 13px;
+  }
+
+  .main__card-wrapper{
+    padding: 0 46px;
+  }
+
+  .product-card__img {
+    min-width: 163px;
+  }
+
+  .product-card:nth-child(2) {
+    display: none;
+  }
+
+  .product-card__discr {
+    width: 100%;
+    font-size: 15px;
+    line-height: 14px;
+    padding-left: 15px;
+    padding-bottom: 4px;
+  }
+
+  .product-card__options {
+    width: 100%;
+  }
+
+  .product-card__price {
+    font-size: 20px;
+    line-height: 19px;
+    padding-left: 15px;
+  }
+
+  .product-card__sel {
+    width: 75px;
+  }
+
+  .product-card__btn {
+    height: 15px;
+    width: 15px;
+  }
+}
+
+@media (max-width: 320px) {
+
+  .heading__hamburger-wrapper {
+    padding: 24px 20px;
+  }
+
+  .main__wrapper {
+    margin-bottom: 38px;
+  }
+
+  .main__text-wrapper {
+    padding: 62px 20px;
+  }
+
+  .main__card-wrapper {
+    padding: 0 20px
+  }
+
+  .product-card {
+    min-width: 330px;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)